ui/widgets/controlgroup.js

```diff
@@ -30,6 +30,7 @@
 		factory( jQuery );
 	}
 }( function( $ ) {
+var removeClassRegex = /ui-corner-([a-z]){2,6}/g;
```

Comment:
If this stays out of the prototype, then it needs to be prefixed with `controlgroup`. The factory goes away during builds and all variables get put in the same scope.